air.toml

```diff
@@ -0,0 +1,14 @@
+[format]
+line-width = 100
+indent-width = 2
+indent-style = "space"
+line-ending = "auto"
+persistent-line-breaks = true
+exclude = [
```

Comment:
This can be removed due to [`bf5a253` (#73)](https://github.com/eitsupi/neo-r-polars/pull/73/commits/bf5a253c1e789fa8019d22ad6c73f33535f991b3)